server/routes/yourCompletedReports/yourCompletedReports.ts

```diff
@@ -1,27 +1,27 @@
 import { Request, Response } from 'express'
 import CompletedAdjudicationsService from '../../services/completedAdjudicationsService'
 import { pageRequestFrom, PageResponse } from '../../utils/pageResponse'
-import { CompletedAdjudicationSummary } from '../../data/CompletedAdjudicationsData'
 import mojPaginationFromPageResponse from '../../utils/pagination'
+import { ReportedAdjudication } from '../../data/ReportedAdjudicationResult'
 
 export default class YourCompletedReportsRoutes {
   constructor(private readonly completedAdjudicationsService: CompletedAdjudicationsService) {}
 
   private renderView = async (
     req: Request,
     res: Response,
-    results: PageResponse<CompletedAdjudicationSummary>
+    results: PageResponse<ReportedAdjudication>
   ): Promise<void> =>
     res.render(`pages/yourCompletedReports`, {
       yourCompletedReports: results,
       pagination: mojPaginationFromPageResponse(
-        results,
+        results.changeIndex(1),
```

Comment:
It took me a while to figure out what this was doing.

I think we can review the pagination abstraction to make this a bit clearer.

Something for later though (not as important as doing other tickets).